.circleci/config.yml

```diff
@@ -11,7 +11,7 @@ workflows:
 jobs:
   unit-test:
     docker:
-      - image: python:3.7.9
+      - image: python:3.10.4
```

Comment:
Why not just use 3.10? I think the patch version need not be pinned down

Reply:
Good point - updated